ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -355,7 +356,7 @@ calcParserModules ghcFlavor = do
           [ "GHC" </> "Parser.hs" | ghcFlavor >= Ghc901] ++
           [ "Parser.hs" | ghcFlavor < Ghc901]
         )
-        
```

Comment:
surprisingly some redundant whitespace has crept in which is getting removed here incidentally